### PR TITLE
Solved Issue #33

### PR DESCRIPTION
**Pull request by marceldobrica**

### CODE REVIEW

**Comment by marceldobrica** [comment]
Also solves issue #32. I have closed them both.

File: .env [hunk 1]
```diff
@@ -43,4 +43,5 @@ MESSENGER_TRANSPORT_DSN=doctrine://default?auto_setup=0
 ###< symfony/mailer ###
 ARTICLES_ON_PAGE=10
 PROGRAMME_MIN_TIME_MINUTES=15
-PROGRAMME_MAX_TIME_MINUTES=360
\ No newline at end of file
+PROGRAMME_MAX_TIME_MINUTES=360
+HTTP_IMPORT_ADDRESS=''
```
Comment (ovidiuenache) [inline .env]: Too vague. A proper name would be EVOZON_WAREHOUSE_BASE_URI.

You can have multiple import addresses and you need to be more specific.

File: src/HttpClient/HttpClientImportPogramme.php [hunk 2]
```diff
@@ -4,26 +4,39 @@
...
+use Symfony\Contracts\HttpClient\Exception\ServerExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
 use Symfony\Contracts\HttpClient\HttpClientInterface;
 
 class HttpClientImportPogramme
 {
-    public string $http_address = 'http://evozon-internship-data-wh.herokuapp.com/api/sport-programs';
-
-    public string $http_protocol = 'GET';
+    public const HTTP_PROTOCOL = 'GET';
```
Comment (ovidiuenache) [inline src/HttpClient/HttpClientImportPogramme.php]: you don't need a constant for this

Symfony\Component\HttpFoundation\Request::METHOD_GET

and by the way, it is a method, not a protocol

**Review by ovidiuenache — CHANGES_REQUESTED** [review]
Please address the feedback above.

File: .env [hunk 3]
```diff
@@ -43,4 +43,5 @@ MESSENGER_TRANSPORT_DSN=doctrine://default?auto_setup=0
 ###< symfony/mailer ###
 ARTICLES_ON_PAGE=10
 PROGRAMME_MIN_TIME_MINUTES=15
-PROGRAMME_MAX_TIME_MINUTES=360
\ No newline at end of file
+PROGRAMME_MAX_TIME_MINUTES=360
+EVOZON_WAREHOUSE_BASE_URI=''
```
Comment (ovidiuenache) [inline .env:47]: Why empty string? It should lead to the API

**Review by ovidiuenache — CHANGES_REQUESTED** [review]
Please address the feedback above.

File: .env (hunk 3 above)
Comment (marceldobrica) [reply .env:47]: Is empty in .env which is on github. Is not empty in .env.local wich is not on github... I think Rares told us we have this possibility too and I wanted to try it.

**Pull request merged**